tedana/utils.py

```diff
@@ -108,29 +142,21 @@ def make_adaptive_mask(data, mask=None, getsum=False, threshold=1):
     # and count # of echos that pass criterion
     masksum = (np.abs(echo_means) > lthrs).sum(axis=-1)
```

Comment:
I think we should raise an error if any echo means are negative, rather than taking the absolute value.

Reply:
I don't think this should be an error, but I don't think we should take the absolute value either. At this point in the processing almost all values should be positive. It is possible that spatial interpolation will make values that are close to zero before less than zero. Those voxels should not be in the mask.

Rather than a error, I could see using `LGR.info` to store more information. At minimum, we should be logging the `lthrs` values for each echo. Maybe we also want to log the max & min values to give a sense of the range of values.

Reply:
I started logging the lthrs, but not the max and min values.